README.md

```diff
@@ -16,3 +59,20 @@ in the inventory. For example, we create an `ethiopia_production` group that cov
 `ethiopia_production` inventory file.
 
 This allows us to centralize the configuration for each deployment target into a single location - `group_vars/<target>`
+
+### Vault ID labels
+
+Ansible supports the assignment of labels to different vault password files. It also allows the label (known as a vault
+ID) to be stored in the header of the encrypted vault file. This allows Ansible to automatically select a vault password
+when decrypting vault files. We use the vault ID feature to simplify vault management in this repository. The following
+touchpoints are involved:
```

Comment:
Let's also add a sample directory to group_vars with a `vault.yml.decrypted.sample`?

Reply:
I have a plan to instead introduce a single generator script like `bin/create` or something that uses a "templates" folder to spin up everything you need. If we had that, do you think we could do away with the "sample" idea?

Reply:
Sounds good.